core/embed/rust/Cargo.toml

```diff
@@ -15,10 +15,18 @@ micropython = []
 protobuf = ["micropython"]
 ui = []
 dma2d = []
+xframebuffer = []
+display_mono = []
+display_rgb565 = ["ui_antialiasing"]
+display_rgba8888 = ["ui_antialiasing"]
 framebuffer = []
 framebuffer32bit = []
 ui_debug = []
 ui_bounds = []
+ui_antialiasing = []
+ui_blurring = []
+ui_jpeg_decoder = ["jpeg"]
+new_rendering = []
```

Comment:
If the `ProgressiveRenderer` is not used, all the code, including `clone_at_bump`, is optimized out. Why do you believe we should introduce such a feature flag?

Reply:
i don't want to blindly trust the linker optimizer on this -- esp. given that in the past we have seen things that are "clearly unused" stay in because the linker optimizer gets tied up in dynamic dispatch features.

in particular, i would like `static-alloc` / `without-alloc` explicitly excluded from bootloader

Reply:
We can't exclude  `static-alloc / without-alloc` because it's used by ProgressiveRenderer and also by DrawingCache in firmware, as well as in bootloaders.